lib/README.md

```diff
@@ -0,0 +1,6 @@
+The files in `lib/` folder automatically generated by Clang.jl, to regenerate the files:
+
+```console
+julia --project=gen -e 'using Pkg; Pkg.instantiate()'
```

Comment:
It's better to commit the `lib/Manifest.toml` file and thus lock the Clang version, because we probably don't want to change how wrapper codes are generated for most of the time.

Reply:
Actually that's one issue I faced rn, which made me to restructure(had to bring them on top) some struct defs and consts in the wrapper.